src/jquery.payment.coffee

```diff
@@ -76,6 +76,14 @@ $.payment.cards = cards = [
     luhn: true
   }
   {
+    type: 'hipercard'
```

Comment:
Let's add the PR comment over here so it's really clear why we are putting hipercard over diners!